db/migrate/20210324105239_create_nomination_emails.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CreateNominationEmails < ActiveRecord::Migration[6.1]
+  def change
+    create_table :nomination_emails, id: :uuid do |t|
+      t.string     :token
+      t.string     :notify_status
+      t.string     :notify_id, null: false
+      t.datetime   :sent_at
+      t.datetime   :opened_at
+      t.references :school, null: false, foreign_key: true, type: :uuid
```

Comment:
Let's also record the email address we sent to here: the fields on the school can change as GIAS is updated